src/main/java/stsjorbsmod/patches/LoadMiscValuePatch.java

```diff
@@ -0,0 +1,33 @@
+package stsjorbsmod.patches;
+
+import com.evacipated.cardcrawl.modthespire.lib.*;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.helpers.CardLibrary;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import javassist.CtBehavior;
+import stsjorbsmod.cards.CustomJorbsModCard;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+public class LoadMiscValuePatch {
+    @SpirePatch(clz = CardLibrary.class, method = "getCopy", paramtypez = {String.class, int.class, int.class})
+    public static class CardLibrary_getCopyPatch {
+        @SpireInsertPatch(locator = CardID_StringEqualsLocator.class, localvars = "retVal")
```

Comment:
I would recommend using a postfix patch for this instead; this insert patch is relying on them never introducing some new special case based on cardID that isn't related to misc, which seems a little fragile.

Postfix patches can't get local vars but they do get a copy of the original return value, which is all you need here:

```java
@SpirePostfixPatch
public static AbstractCard patch(AbstractCard __result, String key, int upgradeTime, int misc) {
    if (misc != 0 && __result instanceof CustomJorbsModCard) {
        ((CustomJorbsModCard) __result).applyLoadedMiscValue(misc);
    }
    return __result;
}
```